packages/ebsi-authorization-client/src/agent/EBSIAuthorizationClient.ts

```diff
@@ -0,0 +1,387 @@
+import { CredentialPayload, IAgentPlugin } from '@veramo/core'
+import { EBSIOIDMetadata, EBSIScope, IRequiredContext, schema, ScopeByDefinition } from '../index'
+import {
+  CreateOAuth2SessionArgs,
+  CreateOAuth2SessionResponse,
+  CreateSIOPSessionArgs,
+  CreateSIOPSessionResponse,
+  ExceptionResponse,
+  GetAccessTokenArgs,
+  GetAccessTokenResponse,
+  GetOIDProviderJwksResponse,
+  GetOIDProviderMetadataResponse,
+  GetPresentationDefinitionArgs,
+  GetPresentationDefinitionResponse,
+  IEBSIAuthorizationClient,
+  InitiateSIOPDidAuthRequestArgs,
+  InitiateSIOPDidAuthRequestResponse,
+} from '../types/IEBSIAuthorizationClient'
+import { uuid } from 'uuidv4'
+
+export class EBSIAuthorizationClient implements IAgentPlugin {
+  readonly schema = schema.IEBSIAuthorizationClient
+  readonly methods: IEBSIAuthorizationClient = {
+    getOIDProviderMetadata: this.getOIDProviderMetadata.bind(this),
```

Comment:
Please prefix them all with 'ebsi', like we do in other agent methods. Remember that these methods will all become available if you include this plugin in an agent. Then a method called "getAccessToken" has no relationship for a developer anymore to EBSI. Hence why we prefix plugin methods, so they make sense to an external developer and are also grouped together

Reply:
Prefixed the methods with ebsiAuth